plugins/MauticCrmBundle/Integration/DynamicsIntegration.php

```diff
@@ -354,6 +356,17 @@ public function pushLead($lead, $config = [])
         return false;
     }
 
+     /**
+      * Convert to UTC date string for API call.
+      */
+    public function getFilterDateUTC(string $paramDate): string
+    {
+        $startDate = new DateTime($paramDate);
+        $startDate->setTimezone(new DateTimeZone('UTC'));
+
+        return sprintf('modifiedon ge %sZ', $startDate->format('Y-m-d\TH:i:s'));
```

Comment:
What language is this?

Reply:
This was copied from the existing message, and IMO out of scope here?
https://github.com/mautic/mautic/pull/13209/files#:~:text=%24oparams%5B%27%24filter%27%5D%20%3D%20sprintf(%27modifiedon%20ge%20%25sZ%27%2C%20substr(%24params%5B%27start%27%5D%2C%200%2C%20%2D6))%3B

Reply:
If it works for you then fine 🙂